src/components/TextInput/BaseTextInput/index.tsx

```diff
@@ -56,12 +60,16 @@ function BaseTextInput(
         autoCorrect = true,
         prefixCharacter = '',
         inputID,
+        markdownEnabled = false,
```

Comment:
Minor thing, but since this is a boolean it should be:

```suggestion
        isMarkdownEnabled = false,
```